test/concentrate/vehicle_position_test.exs

```diff
@@ -374,6 +374,78 @@ defmodule Concentrate.VehiclePositionTest do
       assert Mergeable.merge(first, second) == expected
     end
 
+    test "merge/2 takes the latest non-nil value for crowding" do
+      crowding_1 = %{
+        load: 98,
+        capacity: 100,
+        occupancy_percentage: 0.98,
+        occupancy_status: "FULL"
+      }
+
+      first =
+        new(
+          last_updated: 1,
+          latitude: 1,
+          longitude: 1,
+          crowding: crowding_1
+        )
+
+      crowding_2 = %{
+        load: 2,
+        capacity: 100,
+        occupancy_percentage: 0.02,
+        occupancy_status: "MANY_SEATS_AVAILABLE"
+      }
+
+      second =
+        new(
+          last_updated: 2,
+          latitude: 2,
+          longitude: 2,
+          crowding: crowding_2
+        )
+
+      third_no_crowding =
+        new(
+          last_updated: 3,
+          latitude: 2,
+          longitude: 2,
+          crowding: nil
+        )
+
+      assert %{crowding: ^crowding_2} = Mergeable.merge(first, second)
+      assert %{crowding: ^crowding_2} = Mergeable.merge(second, third_no_crowding)
```

Comment:
**suggestion:** for this and the other test, I think it might make sense to also test that you get the same result with the order of the arguments swapped, since our implementation of merge is supposed to base its priority on `last_updated` rather than just order the values are passed in.